notifier.rst

```diff
@@ -500,10 +500,10 @@ very high and the recipient has a phonenumber::
                 && $recipient instanceof AdminRecipient
                 && null !== $recipient->getPhone()
             ) {
-                return ['sms', 'email'];
+                return ['sms'];
```

Comment:
This example is about returning the sms channel only in case the price is high and the recipient has a phone number.